js-api-spec/importer.test.ts

```diff
@@ -660,6 +661,93 @@ it(
     })
 );
 
+describe('when importer does not return string contents', () => {
+  it('throws an error in sync mode', () =>
+    sandbox(dir => {
+      dir.write({'dir/_other.scss': '// non empty file'});
```

Comment:
It's probably overkill to create a whole sandbox to read from here when you could just construct and return a `Buffer` in memory.